tools/fork-network/src/cli.rs

```diff
@@ -84,7 +81,11 @@ enum SubCommand {
 }
 
 #[derive(clap::Parser)]
-struct InitCmd;
+struct InitCmd {
+    /// If given, the shard layout in this file will be used to generate the forked genesis state
+    #[arg(long)]
+    pub shard_layout_file: Option<PathBuf>,
```

Comment:
In practice, the shard layout file, is it supposed to be a json equivalent to the field `shard_layout` in `genesis`?

Reply:
Yup, just copy pasting it to a new file should work